profiler/src/test/java/com/splunk/opentelemetry/profiler/snapshot/SpecialTraceIds.java

```diff
@@ -0,0 +1,256 @@
+          mapEntry(84, "6b851eb851eb84006aaf238a5e96059e", "95c28f5c28f5c40095aec458603dba39"),
+          mapEntry(85, "6ccccccccccccc006ca07fb7ecfaada7", "947ae147ae147c009342a9dd8f77dc60"),
+          mapEntry(86, "6e147ae147ae14006df6ab9da2960e30", "9333333333333400923d447fc8b10fd5"),
+          mapEntry(87, "6f5c28f5c28f5c006f2d4d597a6fc3d7", "91eb851eb851ec0090fa7eff0fb82c57"),
+          mapEntry(88, "70a3d70a3d70a4006fc7e9a6f4845d4c", "90a3d70a3d70a4008f819f78ef9bc423"),
+          mapEntry(89, "71eb851eb851ec0071b8098256025417", "8f5c28f5c28f5c008e72c4a4db735441"),
+          mapEntry(90, "733333333333340071f57a33efdc6da8", "8e147ae147ae14008df7fb9bef3bf8ce"),
+          mapEntry(91, "747ae147ae147c00740ec06e42dea4fe", "8ccccccccccccc008b99be522e684a28"),
+          mapEntry(92, "75c28f5c28f5c40074d5c2f1d2e4936e", "8b851eb851eb84008aa7035a138f3337"),
+          mapEntry(93, "770a3d70a3d70c0075e1366af3812a76", "8a3d70a3d70a3c00898302ad06cac6d5"),
+          mapEntry(94, "7851eb851eb8500077d0876e4b2354e9", "88f5c28f5c28f4008811849ccec92749"),
+          mapEntry(95, "7999999999999800798aaa2f8e2ff15f", "87ae147ae147b000866bcaf0a87b8931"),
+          mapEntry(96, "7ae147ae147ae00079a5a248e6e3f743", "866666666666680085617b507a531ed7"),
+          mapEntry(97, "7c28f5c28f5c28007b2e4cc6ef77008c", "851eb851eb85200083e4bbf269f228d6"),
+          mapEntry(98, "7d70a3d70a3d70007cedb23eee80aa1d", "83d70a3d70a3d80082c335f22168fedf"),
+          mapEntry(99, "7eb851eb851eb8007e927fc16f5b143d", "828f5c28f5c29000823d6b632e0bd12f"));
+
+  private static Map.Entry<Integer, List<String>> mapEntry(int percentile, String... traceIds) {
+    return Map.entry(percentile, List.of(traceIds));
+  }
+
+  /**
+   * Generates a map of percentiles to trace IDs that will pass the sampling check in {@link
+   * io.opentelemetry.sdk.trace.samplers.TraceIdRatioBasedSampler} when the corresponding
+   * probability is used (e.g., 5th percentile and 0.05).
+   */
+  public static void main(String[] args) {
+    var ranges = createRanges();
+    var traceIds = generateTraceIds(ranges);
+    System.out.println(asJavaCode(traceIds));
+  }
+
+  private static Queue<PercentileRange> createRanges() {
+    var ranges = new ArrayDeque<PercentileRange>();
+    for (int percentile = 1; percentile <= 99; percentile++) {
+      ranges.add(new PercentileRange(percentile, false));
+      ranges.add(new PercentileRange(percentile, true));
+    }
+    return ranges;
+  }
+
+  private static Map<Integer, List<String>> generateTraceIds(Queue<PercentileRange> ranges) {
+    var map = new LinkedHashMap<Integer, List<String>>();
+    while (!ranges.isEmpty()) {
+      var range = ranges.remove();
+      boolean foundTraceIdInRange = false;
+      while (!foundTraceIdInRange) {
+        var lower = new Random().nextLong(range.lower + 1, range.upper);
+        var traceId = TraceId.fromLongs(range.upper, lower);
```

Comment:
might help to have a comment explaining that using `range.upper` here doesn't have a deeper meaning, `TraceIdRatioBasedSampler` doesn't use the first long.